index.js

```diff
@@ -1,11 +1,21 @@
 const express = require("express");
+const bodyParser = require("body-parser");
 const cheerio = require("cheerio");
 const fetch = require("node-fetch");
 const cache = require("memory-cache");
 const cors = require("cors");
+const twitter = require("twitter");
 
 const app = express();
 app.use(cors());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+const twitterClient = new Twitter({
```

Comment:
`Twitter` is undefined.

replace `const twitter = require("twitter");` with `const Twitter = require("twitter");` from line 7

Reply:
Thanks. I fixed it! :)

Reply:
🙌